README.md

```diff
@@ -24,6 +25,7 @@
    * Provide pluggable patterns and loads.
    * Support different client APIs.
    * Deploy, manage and monitor multiple instances from a single entry point.
+   * Support multiple cloud platform due to integration with Cloud Foundry
```

Comment:
I would probably change that `Support multiple cloud platform including AWS and Cloud Foundry` as we currently do not support more than that. Hopefully we will have Azure soon